packages/component-library/src/components/layout/mt-empty-state/mt-empty-state.vue

```diff
@@ -11,6 +11,14 @@
     <p class="mt-empty-state__description">
       {{ description }}
     </p>
+
+    <div v-if="$slots.link" class="mt-empty-state__link">
```

Comment:
I would replace both slots with properties instead. This makes future non-breaking changes easier because they are more strict. And people can't use the slots for unwanted behavior 🙂 

(Just a general tip: `v-if="$slots.link"` is a anti-pattern we discovered in the last years. The v-if didn't get updated when the slot gets filled after the component is mounted. I would prefer removing the v-if and using a CSS ":empty" selector with "display:none". But in this case as mentioned I would replace it with props)

Reply:
FYI: prop naming should match the Figma file: https://www.figma.com/file/Krub3xbG0vMUdoL8vzEZ7l/%E2%98%84%EF%B8%8F-Meteor-Admin-Foundations-%E2%80%93-6.0.0?type=design&node-id=10044-307923&mode=design&t=EkJ5jb3x0gwZ5crD-0

Reply:
Thanks @jleifeld. There will be pretty many things that I need to change/adjust after your feedback. One question, should I group the related properties (props) into a single object, for example 👇🏻  

```shell
mt-empty-state
    icon
    content
        headline*
        description*
        link
    action
```

Thanks for the tip too. I will discover it one day 😉

Reply:
We should try to use the same properties as in the figma file: 

<img width="240" alt="Bildschirmfoto 2024-04-24 um 09 42 46" src="https://github.com/shopware/meteor/assets/14929254/06703f58-1196-46af-a506-98b300fd88ff">

This would makes it easier to react to changes if a designer changes the component in Figma. In some cases it would make sense to modify it a bit. Example: Instead of "Button: boolean" a "Button:string" value would make more sense. Otherwise we would need another prop just for the button content.

Reply:
✔️